common/enclave/cc_data.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "pdo/common/crypto/crypto.h"
+#include "pdo/common/types.h"
+
+class cc_data
+{
+private:
+    pdo::crypto::sig::PublicKey verification_key_;
+    pdo::crypto::sig::PrivateKey signature_key_;
+    pdo::crypto::pkenc::PublicKey encryption_key_;
+    pdo::crypto::pkenc::PrivateKey decryption_key_;
+    pdo::crypto::pkenc::PublicKey cc_encryption_key_;
+    pdo::crypto::pkenc::PrivateKey cc_decryption_key_;
```

Comment:
```suggestion
    pdo::crypto::sig::PublicKey verification_key_; // enclave_vk
    pdo::crypto::sig::PrivateKey signature_key_; // enclave_sk
    pdo::crypto::pkenc::PublicKey encryption_key_;
    pdo::crypto::pkenc::PrivateKey decryption_key_;
    pdo::crypto::pkenc::PublicKey cc_encryption_key_; // chaincode_ek
    pdo::crypto::pkenc::PrivateKey cc_decryption_key_; // chaincode_dk
```

Reply:
For FPC chaincode state encryption we defined to use a symmetric key `sgx_aes_gcm_128bit_key_t sek` as denoted in `interfaces.md` Is this asymmetric key here intended?

Reply:
aren't you talking here about a different key?  We have the chaincode public encryption key for the chaincode invocation requests and we have, separately, the chaincode state encryption key?  But obviously we need both, so one seems missing.  That said, from _cc_ state perspective, only these two are chaincode and not enclave specific.  So thinking more long-term sooner or latter you would want two different states; that's also related to why `initEnclave'/`ecall_init` only dealt with enclave keys and the chaincode secrets were created with a separate `generateCCKeys`/`ecall_generate_cc_keys` flows ...

Reply:
added comments

Reply:
ok but what is `encryption_key_` and `decryption_key_`?

Reply:
That would be the enclave-specific de/encryption key.

Reply:
this key actually wouldn't be used in MVP, only needed during key-management?

Reply:
right

Reply:
> ok but what is `encryption_key_` and `decryption_key_`?

these keys are required for our key exchange protocol as specified in [fpc-key-dist.puml](https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/docs/design/fabric-v2%2B/fpc-key-dist.puml). In the diagram we refer to them as `enclave_ek = encryption_key_` and `enclave_dk = decryption_key_`.

@bvavala please also add this missing comment :)